Global/GLOBAL_VAR.gd

```diff
@@ -13,8 +13,17 @@ var MODIFIER_KEYS: Dictionary = {
 # Keys can be accessed with the syntax: GlobalVar.MODIFIER_KEYS.ADD_PERMANENT
 # this is useful if we need to change the reference value in multiple places
 
-## A list of all the possible events
-var EVENTS_CLASSIFICATION: Array[Resource] = [EventMob, EventRandom, EventShop, EventHeal, EventDialogue]
+## A list of all the possible events, ordered the same as GlobalEnums.EventType
+var EVENTS_CLASSIFICATION: Array[Resource] = [EventRandom, EventHeal, EventMob, EventShop, EventDialogue]
```

Comment:
Hm I thought it'd be better if things were consistent across classes, but I could change it to a Dictionary instead, so that it is linked without needing to keep the order

Reply:
I'm thinking we might be able to remove this maybe :thinking: because we have the event types with the global enums already
That might require changing the places that need this array, so maybe out of scope for this PR;
But at the same time, since you are already working on events, might as well